src/referentielijsten/conf/base.py

```diff
@@ -284,6 +286,10 @@
             "level": "INFO",
             "propagate": True,
         },
+        "mozilla_django_oidc": {
+            "handlers": ["project"],
```

Comment:
Let's make logging to console possible
```suggestion
            "handlers": ["project"] if not LOG_STDOUT else ["console"],
```